src/MCPClient/lib/clientScripts/getAipStorageLocations.py

```diff
@@ -22,6 +19,8 @@ def get_aip_storage_locations(purpose):
 
 
 if __name__ == '__main__':
+    logger = get_script_logger("archivematica.mcp.client.getAipStorageLocations")
```

Comment:
This is how I'm looking at handling the logs for client scripts. This function sets up a root logger for `archivematica` as well as a logger object for the specific script, and returns the latter. All of the scripts log to the same `client_scripts` log file, with the filename being at the beginning of the message to make it easier to tell where logging messages are coming from. An example:

> getAipStorageLocations.py: INFO     archivematica.common.get_location: Storage locations returned: [{'uuid': '516b19e7-cfd9-4c44-8f99-b1fe27bf390e', 'pipeline': ['/api/v2/pipeline/7600ac20-4f62-4de6-943d-7a2e8a29aa5a/', '/api/v2/pipeline/f61d9185-cd18-4d91-9c91-8087fbfe88bd/', '/api/v2/pipeline/294e6779-56f4-4468-8d9b-94b311d12eac/', '/api/v2/pipeline/be01d080-139c-4eec-9bfe-d037c70e160a/'], 'used': '4887384', 'description': 'Store AIP in standard Archivematica Directory', 'space': '/api/v2/space/3dcb3b17-790d-44e3-b948-943a62c6a80c/', 'enabled': True, 'quota': None, 'relative_path': 'var/archivematica/sharedDirectory/www/AIPsStore', 'purpose': 'AS', 'path': '/var/archivematica/sharedDirectory/www/AIPsStore', 'resource_uri': '/api/v2/location/516b19e7-cfd9-4c44-8f99-b1fe27bf390e/'}]

When we add logging within the client scripts themselves, I was thinking we'd pass a `logger` keyword argument into the `main` function (with the default being a logger with a null handler), rather than relying on a file-wide logger object like in library code.

Right now the log level is hardcoded, but we could look at making it configured in the MCPClient settings file and have the scripts fetch the level to pass into this function.